system/Helpers/kint_helper.php

```diff
@@ -16,7 +16,9 @@
         /**
          * Prints a Kint debug report and exits.
          *
-         * @param array ...$vars
+         * @param array $vars
+         *
+         * @return void
```

Comment:
```suggestion
         * @return void
         * @phpstan-return never
```

Reply:
Please apply my suggested change.
The true return type is not `void`, but `never`.
See https://phpstan.org/writing-php-code/phpdoc-types#bottom-type
https://www.php.net/manual/en/language.types.never.php